src/Components/Panel/Panel.scss

```diff
@@ -30,45 +32,114 @@
   align-items: center;
 }
 
+.panel--body {
+  > *:last-child {
+    margin-bottom: 0;
+  }
+}
+
 .panel--footer {
   border-radius: 0 0 $cf-radius $cf-radius;
   background-color: rgba($g0-obsidian, 0.1);
 }
 
+/*
+  Dismissable Panel
+  ------------------------------------------------------------------------------
+*/
+
+$panel--dismiss-button: 30px;
+
+.panel__dismissable {
+  position: relative;
+
+  > .panel--header {
+    padding-right: 40px;
+  }
+}
+
+.panel--dismiss {
+  position: absolute;
+  background-color: transparent;
+  border: 0;
+  transition: opacity 0.25s ease;
+  opacity: 0.4;
+  outline: none;
+
+  &:before,
+  &:after {
+    content: '';
+    height: $cf-border;
+    border-radius: $cf-border / 2;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+  }
+
+  &:before {
+    transform: translate(-50%, -50%) rotate(45deg);
+  }
+
+  &:after {
+    transform: translate(-50%, -50%) rotate(-45deg);
+  }
+
+  &:hover {
+    cursor: pointer;
+    opacity: 1;
+  }
+}
+
 /*
   Size Modifiers
   ------------------------------------------------------------------------------
 */
-@mixin panelSizeModifier($fontSize, $fontWeight, $padding) {
+@mixin panelSizeModifier($fontSize, $fontWeight, $padding, $dismissSize) {
```

Comment:
From the GIF, it felt like the line height for the `xs` size was a bit large.

Reply:
Ah yeah I know watcha mean. Currently the children of `PanelBody` are not affected by the size of `Panel` but I was thinking about having some styles to affect typographic elements within panels to respond to panel sizing

Reply:
That would be rad, but nbd either way.